app/models/event_type.rb

```diff
@@ -10,6 +10,7 @@ class EventType < ActiveRecord::Base
 
   alias_attribute :name, :title
 
+  # If LENGTH_STEP > 60 the public schedule must be changed
```

Comment:
In fact, we need the step to be divisor of 60 to display the schedule properly. How can I force that the step is a divisor of 60? I can add a test, but I am not pretty sure if that is what you mean.